packages/app/package.json

```diff
@@ -51,6 +51,7 @@
     "@k-phoen/backstage-plugin-grafana": "^0.1.22",
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.9.1",
+    "backstage-plugin-dask-onboarding": "^0.1.3",
```

Comment:
I tried to utilize the env variable for the backstage backend in this package `APP_CONFIG_backend_baseUrl`, let me know if I should use this one or another url.

Reply:
If that URL contains what you need then that is perfectly fine to use. Most other plugins use it as well :slightly_smiling_face: